src/styles/stars.scss

```diff
@@ -0,0 +1,27 @@
+@use './variables.scss' as *;
+
+.stars {
+  width: 96px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  @for $i from 1 through 5 {
+    &__star-#{$i} {
+      width: 16px;
+      height: 16px;
```

Comment:
The width and height of each star are hardcoded as 16px. Consider using variables from your variables.scss file for consistency and easier adjustments.